manifests/piped/values.yaml

```diff
@@ -78,6 +78,8 @@ secret:
     fileName: datadog-application-key
     data: ""
 
+containerEnv: {}
```

Comment:
nit: `envs: []`
As you see we are defining all fields of the container at the top level like `resources` so I think we can name it as `envs` without using `container` prefix. And this is an array so we should use `[]` instead of `{}`.